mistralrs-core/src/xlora_models/gemma.rs

```diff
@@ -736,6 +736,7 @@ impl NormalModel for XLoraModel {
     fn device(&self) -> &Device {
         &self.device
     }
+    // BUG?: This is the only x-lora model with `is_xlora() -> false`
```

Comment:
*Yes*! Thanks for catching that. As a long term and safer solution, maybe we can get this information from the `ModelKind` instead of using hardcoded values? Just an idea.

Reply:
💯 I was thinking that made sense too.

Should `is_xlora()` be `true` here when the `AdapterKind` is `Lora` instead of `XLora`? That was one of my concerns about how these two are treated since presently these would return `true` for `Lora` kind.